lib/lrama/grammar/parameterizing_rule/resolver.rb

```diff
@@ -13,17 +13,22 @@ def add_parameterizing_rule(rule)
           @rules << rule
         end
 
-        def find(token)
+        def find_rule(token)
           select_rules(token).last
         end
 
+        def find_inline(token)
+          @rules.select { |rule| rule.name == token.s_value && rule.is_inline }.last
+        end
+
         def created_lhs(lhs_s_value)
           @created_lhs_list.reverse.find { |created_lhs| created_lhs.s_value == lhs_s_value }
         end
 
         private
 
         def select_rules(token)
+          rules = select_not_inline_rules
           rules = select_rules_by_name(token.rule_name)
```

Comment:
It seems the result o line 31 is not used by line 32 and overwrote by line 32. Is something wrong?

Reply:
😨 My apologies... I updated it.